README.md

```diff
@@ -73,11 +73,24 @@ Or, if the variable is someplace a bit more complex, like a loop:
 {% endfor %}
 ```
 
+### Lazy loading images
+
+For pages showing a large number of avatars, you may want to load the images lazily.
+
+```liquid
+{% avatar hubot lazy=true %}
```

Comment:
Nice. Is there an advantage to making the flag per-tag versus one global flag? Would users want some avatars to be lazy loaded and others to be loaded in-line?

Reply:
Umm, I don't feel super strongly either way. This seems more flexible, so I would err on this side.

Reply:
👍 Easy enough to make it global later if people want it.